.circleci/config.yml

```diff
@@ -90,6 +90,10 @@ jobs:
             # Set the version
             if [[ $CIRCLE_TAG ]]; then
                 export VERSION=$CIRCLE_TAG
+            # for main branches, can't add the commit hash since it's not a valid format for publishing
+            elif [[ $CIRCLE_BRANCH == "main" ]]; then
```

Comment:
I think you can use `pipeline-branch` (defined [here](https://github.com/datagouv/hydra/blob/6d49cda6224eae3fb3405eb905abbdbd234c4e71/.circleci/config.yml#L11-L13)) instead of "main"

Reply:
You mean `publish-branch` right? Yes good idea

Reply:
Yes indeed, my bad!

Reply:
Done